content/docs/main/resiliency/retry/about.md

```diff
@@ -0,0 +1,18 @@
+---
+title: About
+weight: 10
+description:
+---
+
+A retry is the number of times a request is retried if it fails. This setting can be useful to avoid your apps from failing if they are temporarily unavailable. With retries, calls are retried a certain number of times before they are considered failed. Retries can enhance your app's availability by making sure that calls don’t fail permanently because of transient problems, such as a temporarily overloaded service or network.
+
+## Configuration options
+
+You can configure retries by using a Kubernetes Gateway API-native configuration or a {{< reuse "docs/snippets/kgateway.md" >}}{{< reuse "docs/snippets/trafficpolicy.md" >}} as shown in the following table.
+
+
+| Type of timeout| Description | Configured via | Attach to | 
+| -- | -- | -- | --- | 
+| [Request retries]({{< link-hextra path="/resiliency/retry/retry/" >}}) | Specify the number of times and duration for the gateway to try a connection to an unresponsive backend service. | <ul><li>HTTPRoute</li><li>{{< reuse "docs/snippets/trafficpolicy.md" >}} </li></ul>| <ul><li>HTTPRoute</li><li>HTTPRoute rule</li><li>Gateway listener</li></ul> | 
```

Comment:
Does the Gateway API support attaching to a Gateway listener? for other policies that are inline the HTTPRoute, it doesn't and that's why we end up supporting it in the TrafficPolicy.